e2e_deployment_files/long_haul_deployment.template.windows.json

```diff
@@ -21,25 +21,19 @@
           "edgeAgent": {
             "type": "docker",
             "settings": {
-              "image": "<Container_Registry>/microsoft/azureiotedge-agent:<Build.BuildNumber>-linux-<Architecture>",
```

Comment:
This looks weird to me; why windows deployment file uses linux image.

Reply:
Whatever was happening here was standard practice for 1.0.8. If I had to guess, at the time of the first 1.0.8 release windows images were not available and no one has ported an update until now.